cpp/arcticdb/python/python_handlers.cpp

```diff
@@ -0,0 +1,24 @@
+#include <python/python_handlers.hpp>
```

Comment:
@mehertz: what is Man Group's policy regarding BSL headers? Must they always be present in header and source files?

Reply:
@jjerphan what is a "BSL header"?

Reply:
BSL stands for "Business Software License", [the license used by ArcticDB](https://github.com/man-group/ArcticDB/blob/master/LICENSE.txt).

Generally each file has an header in their first lines. See for instance:
https://github.com/man-group/ArcticDB/blob/bf9a8310bfcb7aa484778fd1ba764a2250f9b429/cpp/arcticdb/pipeline/frame_utils.hpp#L1-L6